resources/templates/account_password.html

```diff
@@ -9,9 +9,13 @@
 <div class="info-panel">
   <div>
     <input type="password" name="passphrase1" placeholder="Secure password" />
+    <a>
+      <img id="eye" class="icon-link" src="{{ static_url('images/eye.svg') }}" />
+    </a>
   </div>
   <div>
     <input type="password" name="passphrase2" placeholder="Please confirm" />
+    &emsp;
```

Comment:
Why is that needed?

Reply:
As i remember it now since I dont put `a ` tag (of the eye img) for the second input i require a tiny amount of spacing there otherwise the inputs get misaligned

Reply:
Now I see, thanks!